src/modules/poolModule.ts

```diff
@@ -606,8 +607,8 @@ export class PoolModule implements IModule {
     while (true) {
       const typeArguments = [params.coinTypeA, params.coinTypeB]
 
-      const tx = new TransactionBlock()
-      const args = [tx.object(params.pool_id), tx.pure(start, 'u64'), tx.pure(limit.toString(), 'u64')]
+      const tx = new Transaction()
+      const args = [tx.object(params.pool_id), tx.pure(bcs.vector(bcs.Address).serialize(start)), tx.pure.u64(limit)]
```

Comment:
` tx.pure(start, 'u64'),` didn't make a lot of sense to be based on start being a list of addresses.  This should be double checked